src/Makefile

```diff
@@ -155,7 +155,7 @@ endif
 
 ### 3.1 Selecting compiler (default = gcc)
 F :=
-CXXFLAGS += -Wall -Wcast-qual -fno-exceptions -std=c++11 $(EXTRACXXFLAGS) $(F)
+CXXFLAGS += -Wall -Wcast-qual -fno-exceptions -std=c++11 $(EXTRACXXFLAGS)$(F)
```

Comment:
can you explain what that approach with `$(F)`

Reply:
apologize, indeed I'm in disaster for 3 days; my Windows crashed and so did my Linux and still try to make Iso bootable usb luckily a family member keep generous giving me online inet free to google a solution.
Just an optional flag for compiler

- simply ignore it, then just nothing
- if one opt to add it e.g.
  make build ARCH=x86-64-modern F=-flto
  and he is on Mingw64 system in that he's sure it fully works in building, executing it, then it'll build with link time optimization
- I personally find useful to ensure metaprogramming correctness by check F=-Werror=vla
 so there'll be no "fake" builtin array if it compiles perfectly

Reply:
can't we do this already with `CXXFLAGS='-Werror=vla' make -j ARCH=.....` ?

Reply:
you're correct, the same, is simpler so easier, faster to user/cpu